tox.ini

```diff
@@ -22,6 +22,8 @@ conda_deps =
     pytest-mock
     pytest-xdist
     scipy >= 1.2.1
+    pathos
```

Comment:
Do we really use pathos? Otherwise we should remove it here and in the other places.